app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
Yes and add the color picker as input field in the form as I asked in one of my previous comments.